kstreams/backends/kafka.py

```diff
@@ -54,7 +54,7 @@ class Kafka(BaseModel):
     bootstrap_servers: List[str] = ["localhost:9092"]
     security_protocol: SecurityProtocol = SecurityProtocol.PLAINTEXT
 
-    ssl_context: Optional[ssl.SSLContext]
+    ssl_context: Optional[ssl.SSLContext] = None
```

Comment:
Optional is no longer enough to mean not required
https://docs.pydantic.dev/latest/migration/#required-optional-and-nullable-fields